README.md

```diff
@@ -22,11 +22,23 @@ Pre-build images are available on [DockerHub](https://hub.docker.com/r/existdb/e
 To download the image run:
 ```bash
 docker pull existdb/existdb:latest
-docker run -it -d -p 8080:8080 -p 8443:8443 existdb/existdb:latest
 ```
 
-You can now access eXist via [localhost:8080](localhost:8080) in your browser.
-Try to stick with matching internal and external port assignments, to avoid unnecessary reloads and connection issues.
+once the download is complete, you can run the image
+```bash
+docker run -it -d -p 8080:8080 -p 8443:8443 --name exist existdb/existdb:latest
+```
+
+What does this do?
+
+*   `-it` allocates a TTY and keeps STDIN open
```

Comment:
I fear we are just reproducing the Docker docs here. If this is meant to be a smaller example for first-time Docker eXist-db users, perhaps we should add some more friendly explanation. Many eXist-db users won't have a clue what a TTY or STDIN is.

e.g.

* `-it` allocates a TTY and keeps STDIN open. This allows you to interact with the running Docker container via your console.

Not sure what to write for `-d` as I have never used it with `it`.

* `-p` maps TCP ports from the container to your host machine (we recommend sticking with matching pairs). This allows you to connect to the eXist-db Web Server running in the Docker container. As described, port 8080  for HTTP and port 8433 for HTTPS.

You get the idea ;-)